pkg/kubeapiserver/options/authorization.go

```diff
@@ -68,6 +71,13 @@ func (s *BuiltInAuthorizationOptions) AddFlags(fs *pflag.FlagSet) {
 		"authorization-webhook-cache-unauthorized-ttl", s.WebhookCacheUnauthorizedTTL,
 		"The duration to cache 'unauthorized' responses from the webhook authorizer.")
 
+	fs.BoolVar(&s.WebhookUseServiceResolution,
+		"authorization-webhook-use-service-resolution", s.WebhookUseServiceResolution,
+		"If set, the authorizer webhook code will resolve URLs pointing to hosts of the form "+
+			"https://servicename.namespace.svc:port/somewhere to appropriate endpoint IP addresses. "+
```

Comment:
keep the indention as in the other flags please

Reply:
Other flags use `, ""+` to force the formatting.

https://play.golang.org/p/nU_aZ7Wd9M

Reply:
Done